tempest/api/workloadmgr/workload_reassign/test_workload_reassign_same_project_same_user.py

```diff
@@ -0,0 +1,170 @@
+
+
+    @decorators.idempotent_id('9fe07175-912e-49a5-a629-5f52eeada4c9')
+    @decorators.attr(type='workloadmgr_api')
+    def test_workload_reassign_same_project_same_user(self):
+        try:
+
+            tests = [['tempest.api.workloadmgr.workload_reassignment.create_workload_and_snapshot',
+                      0],
+                     ['tempest.api.workloadmgr.workload_reassignment.assign_tenant1_and_user1',
+                      0],
+                     ['tempest.api.workloadmgr.workload_reassignment.reassign_same_tenant1_and_user1',
+                      0]]
+
+            reporting.add_test_script(tests[0][0])
+
+            ### Create vm and workload ###
+            LOG.debug("Create VM")
+
+            self.created = False
+            vm_id = self.create_vm(vm_cleanup=True)
+            LOG.debug("\nVm id : {}\n".format(str(vm_id)))
+
+            #Create the trust
+            trust_id = self.create_trust(tvaultconf.trustee_role)
+            if trust_id:
+                reporting.add_test_step("Create user trust on project", tvaultconf.PASS)
+            else:
+                LOG.error("Create user trust on project failed.")
+                raise Exception("Create user trust on project")
+
+            LOG.debug("Create workload")
+            workload_id = self.workload_create(
+                [vm_id], tvaultconf.parallel, workload_cleanup=True)
+            LOG.debug("Workload ID: " + str(workload_id))
+
+            if(workload_id is not None):
+                self.wait_for_workload_tobe_available(workload_id)
+                if(self.getWorkloadStatus(workload_id) == "available"):
+                    reporting.add_test_step("Create workload", tvaultconf.PASS)
+                else:
+                    LOG.error("Failed to get workload status as available for workload ID: " + str(workload_id))
+                    raise Exception("Create workload")
+            else:
+                LOG.error("Failed to create workload.")
+                raise Exception("Create workload")
+
+            #take a full snapshot of it.
+            snapshot_id = self.create_snapshot(workload_id, is_full=True)
+
+            self.wait_for_workload_tobe_available(workload_id)
+            snapshot_status = self.getSnapshotStatus(workload_id, snapshot_id)
+
+            if (snapshot_status == "available"):
+                LOG.debug("Full snapshot created.")
+                reporting.add_test_step("Create full snapshot", tvaultconf.PASS)
+                tests[0][1] = 1
+            else:
+                LOG.error("Full snapshot creation failed.")
+                raise Exception("Create full snapshot")
+
+
+            #assign tenant1 and user1 to workload created.
+            reporting.test_case_to_write()
+            reporting.add_test_script(tests[1][0])
+            LOG.debug("Assign tenant and user to workload ID: " + str(workload_id))
+
+            ### copy tenant1 and user1
+            tenant_id = CONF.identity.tenant_id
```

Comment:
These values are already the default values assigned. No need to explicitly assign same values again.

Reply:
By mistake I changed it to tenant_id and user_id instead of tenant_id_1 and user_id_1. Please check updated code. I verified the new changes as well.